frontend/src/components/ShootWorkers/VolumeType.vue

```diff
@@ -61,15 +62,58 @@ export default {
     volumeTypes: {
       type: Array,
       default: () => []
+    },
+    cloudProfileName: {
+      type: String
     }
   },
   data () {
     return {
-      validationErrors,
-      valid: undefined
+      valid: undefined,
+      internalIops: undefined
```

Comment:
Why do we need the `internalIops`. We already have an eventHandler for @input.

Reply:
Actually, the whole point of the internalIops has a reactivity background. We cannot operate on `worker.providerConfig.volume.iops` as the path may not exists initially and therefore validation does not work correctly. If you can show me a way that makes this work without the ìnternal` thingy, please let me know.